src/components/application_manager/src/hmi_state.cc

```diff
@@ -142,13 +159,22 @@ VideoStreamingHmiState::VideoStreamingHmiState(
 
 mobile_apis::VideoStreamingState::eType
 VideoStreamingHmiState::video_streaming_state() const {
+  return std::max(parent()->video_streaming_state(),
```

Comment:
In practice, this shouldn't be a problem only because if how I set it up (as is the case with why the check wasn't there prior to my changes). That said, I could add this check just for safety